relay-server/src/actors/processor.rs

```diff
@@ -2337,6 +2337,15 @@ impl EnvelopeProcessorService {
         if all_modules_enabled {
             // Extract tags to add to this span as well
             let shared_tags = span::tag_extraction::extract_shared_tags(event);
+            transaction_span.sentry_tags = Annotated::new(
+                shared_tags
+                    .clone()
+                    .into_iter()
+                    .map(|(k, v)| (k.sentry_tag_key().to_owned(), Annotated::new(v)))
+                    .collect(),
+            );
```

Comment:
Not related to this PR -- it seems the top-level transaction, converted to a span, only has the shared tags and not span-specific tags. Is this expected?

Reply:
Good observation! It's unclear which span-specific tags a transaction should have. We will come back to this in https://github.com/getsentry/relay/issues/2494.